renderer/src/components/CharacterKillsCard.jsx

```
@@ -184,93 +254,73 @@
+        </div>
+      )}
       
       <div className="p-4">
         <div className="grid grid-cols-2 gap-3 mb-4">
           <button
             onClick={fetchAllCharacterKills}
             disabled={loading}
             title="Load Kill Data"
-            className="p-3 bg-indigo-600 text-white rounded-full hover:bg-indigo-700 transition flex items-center justify-center disabled:opacity-50"
+            className="p-3 bg-indigo-600 text-white rounded-lg hover:bg-indigo-700 transition flex items-center justify-center disabled:opacity-50"
           >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)